pandas/tests/generic/test_generic.py

```diff
@@ -424,6 +434,14 @@ def test_sample(self):
         weights_with_None[5] = 0.5
         self._compare(o.sample(n=1, axis=0, weights=weights_with_None), o.iloc[5:6])
 
+    def test_sample_upsampling_without_replacement(self):
+        df = pd.DataFrame({"A": list("abc")})
```

Comment:
can you add the issue number as a comment

Reply:
I removed the hash